plugins/generator-1.19.4/forge-1.19.4/templates/json/rangeditem.json.ftl

```diff
@@ -2,7 +2,22 @@
     "parent": "item/generated",
     "textures": {
       "layer0": "${modid}:item/${data.texture}"
-    },
+    }

```

Comment:
It would be more obvious and less changes if you left comma here and just added another one at the end of overrides array closure ]

Reply:
:heavy_check_mark: Fixed